Cloud Application Manager/Deploying Anywhere/using-compute-instances.md

```diff
@@ -0,0 +1,68 @@
+"date": "11-05-2019",
+"author": "Sharon Wang",
+"attachments": [],
+"contentIsHTML": false
+}}}
+
+**In this article:**
+
+* [Overview](#overview)
+* [Audience](#audience)
+* [Prerequisites](#prerequisites)
+* [Create a Provider](#create-a-provider)
+* [Register a Server](#register-a-server)
+* [Contacting Cloud Application Manager Support](#contacting-cloud-application-manager-support)
+
+### Overview 
+
+This article is meant to assist users of Cloud Application Manager trying to register their stand-alone/physical machines to CAM. You can bring your own infrastructure to Cloud Application Manager by installing the Cloud Application Manager agent and register it to Compute Instances Provider. This will enable lifecycle of the instance through Cloud Application Manager like with any other natively-deployed instance.
+
+### Audience
+
+All users of Cloud Application Manager who wants to register their stand-alone machines to Compute Instances Provider.
+
+### Prerequisites
+
+* An active **Cloud Application Manager** account
+* Any infrastructure capable of running Cloud Application Manager Agent.
+
+
+### Create a Provider
+
+The preferred method to register any infrastructure in CAM, is to register it into a provider, instead of a stand-alone instance. To learn more about registering your servers as a stand-alone instance, please review the [Deploying Anywhere](./deploying-on-anyinfra.md)
+
+To create a new provider, select **Compute Instances** from the Provider list and give it a name, then click on **Save**
+
+![centurylink-add-provider.png](../../images/cloud-application-manager/deploying-anywhere/compute-instances/compute_instances_1.png)
+
+**Note**: Unlike other provider types, this one does not require any synchronization process and it does not support deployment policies either, since there is no direct deployment support on it.
+
+### Register a Server
+You can launch boxes on any infrastructure by running the Cloud Application Manager agent. The agent is required to execute box variables and scripts and run lifecycle operations. Once the provider is created, users can register VMs or physical machines to this Provider by clicking **Self-Register Scripts**
+
+![register-a-server-1.png](../../images/cloud-application-manager/deploying-anywhere/compute-instances/compute_instances_2.png)
+
+There are 2 scripts for Windows and Linux OS. In this article, we use Windows server as an example:
+![register-a-server-2.png](../../images/cloud-application-manager/deploying-anywhere/compute-instances/compute_instances_3.png)
+
```

Comment:
Please, explain briefly about the proxy support in the code snippets (probably it is already explained in the self-registering article, but at least mention it here briefly)